evml/torch/mc_dropout.py

```diff
@@ -1,9 +1,8 @@
 import sys
 import numpy as np
 import torch
-import torch.nn as nn
 import torch.nn.functional as F
-from evml.class_losses import relu_evidence
+from evml.torch.class_losses import relu_evidence
 
```

Comment:
I don't know that we necessarily need to point to stack, but most of this code came from the given answer here: https://stackoverflow.com/questions/63285197/measuring-uncertainty-using-mc-dropout-on-pytorch

Reply:
Added!